setup.sh

```diff
@@ -36,6 +52,9 @@ if [ ! -f /etc/VMInit.py ]; then
         echo "Copying files failed"
```

Comment:
I don't think this is needed any more. There are exit commands on every error check.